src/howard/howard/templates/marion/certificates/realisation.html

```diff
@@ -63,8 +63,8 @@ <h1>CERTIFICAT DE REALISATION</h1>
           </div>
           <div class="dates">
             qui s'est déroulée du
-            <span>{{ course.session.date.from }}</span> au
-            <span>{{ course.session.date.to }} </span>
+            <span>{{ course.session.date.start }}</span> au
+            <span>{{ course.session.date.end }} </span>
```

Comment:
Once deserialized (thanks to pydantic) and typed, one should use Django's `date` filter :tada: 

```suggestion
            <span>{{ course.session.date.end | date }} </span>
```